src/ext/java/org/opentripplanner/ext/transmodelapi/model/siri/et/EstimatedCallType.java

```diff
@@ -288,6 +289,22 @@ public static GraphQLObjectType create(
           .dataFetcher(environment -> ((TripTimeOnDate) environment.getSource()).getTrip())
           .build()
       )
+      .field(
+        GraphQLFieldDefinition
+          .newFieldDefinition()
+          .name("datedServiceJourney")
+          .type(datedServiceJourneyType)
+          .dataFetcher(environment -> {
+            TripTimeOnDate tripTimeOnDate = environment.getSource();
+            return GqlUtil
+              .getRoutingService(environment)
```

Comment:
This should use the datedServiceJourney Helper class

Reply:
Done